src/scripts/services/testing-service.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable functional/no-return-void */
```

Comment:
We can move this up a level and call it `data-providers.ts`. It isn't a service in the sense that we use the word.

Reply:
So move it into the scripts directory?

Reply:
When I try to move it into the scripts directory and update the imports, running any commands causes the operation to fail because jest isn't being recognised.

Reply:
`data-providers.test.ts` ?

Reply:
Then it means that running `npm run test` would fail as every test file needs to have a test in it

Reply:
Oh right.
1. Put in a `__fixtures__` directory, named `data-providers.ts`.
2. Make sure tsconfig is ignoring that directory.

Reply:
Put `_fixtures_` in the scripts directory i.e. `scripts/_fixtures_/data-providers.ts`?

And I presume step 2 involves adding `_fixtures_` to the exclude list.

Reply:
Yes. Two underscores on either side.